app/auth/defra-id/validate-token.js

```diff
@@ -1,7 +1,7 @@
 const { getScopes } = require('./get-scopes')
 
 const validateToken = (decoded, _request, _h) => {
-  return { isValid: true, credentials: { scope: getScopes(decoded.roles), name: `${decoded.firstName} ${decoded.lastName}` } }
+  return { isValid: true, credentials: { scope: getScopes(decoded.roles), name: `${decoded.firstName} ${decoded.lastName}`, crn: decoded.contactId } }
```

Comment:
Adding the CRN to the `credentials` property will mean it's available in our route handlers without needing to decode the cookie.

Similar to what we've done with `name`.